tools/dotnet-deb-tool/project.json

```diff
@@ -0,0 +1,25 @@
+{
+  "version": "1.0.0-tool-*",
+  "buildOptions": {
+    "emitEntryPoint": true,
+    "compile": {
+      "exclude": [ "tool/**/*" ]
+    }
+  },
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0-rc3-004308"
+    }
+  },
+
+  "packInclude": {
```

Comment:
Yes it gives a warning, do you happen to know if there exists some documentation on the `packOptions`?